backend/loan/views.py

```diff
@@ -16,8 +17,8 @@ def loan_list(request):
     if request.method != 'GET' and request.method != 'POST':
         return HttpResponseNotAllowed(['GET', 'POST'])
 
-    if not request.user.is_authenticated:
-        return HttpResponse(status=401)
+    # if not request.user.is_authenticated:
+    #    return HttpResponse(status=401)
```

Comment:
Same here ^

Reply:
This file seems to cause merge conflicts (and changes to it are unnecessary), so I would consider just reverting this file.